dlt-database/entity/0001-init_db/User.ts

```diff
@@ -0,0 +1,39 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, OneToMany, JoinColumn } from 'typeorm'
+
+import { Account } from './Account'
+
+@Entity('users', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
+export class User extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({
+    name: 'gradido_id',
+    length: 36,
+    nullable: true,
+    unique: true,
```

Comment:
yes the gradidoId has to be unique, but only belonging to one community.
Overall communities there could exist duplicate gradidoId for different users. 
The uniqueness must exist together with the communityUUID and the gradidoId for one user.